data/journal_check/bug_refs.json

```
@@ -692,5 +693,26 @@
             "opensuse": ["Tumbleweed"]
         },
         "type": "bug"
+    },
+    "boo#1227900": {
+        "description": "Speculative Return Stack Overflow: IBPB-extending microcode not applied!|Speculative Return Stack Overflow: WARNING: See https://kernel.org/doc/html/latest/admin-guide/hw-vuln/srso.html for mitigation options.",
+        "products": {
+            "opensuse": ["Tumbleweed"]
+        },
+        "type": "bug"
```

Host issue, not SUT issue.

```suggestion
        "type": "ignore"
```

---

fixed